sycl/doc/extensions/experimental/sycl_ext_intel_esimd/sycl_ext_intel_esimd.md

```diff
@@ -831,8 +831,8 @@ simd<float, 16> vb;
 simd<float, 16> vc;
 
 __asm__("add (M1, 16) %0 %1 %2"
-                    : "=rw"(vc.data_ref())
-                    : "rw"(va.data()), "rw"(vb.data()));
+                    : "=r"(vc.data_ref())
+                    : "r"(va.data()), "r"(vb.data()));
 ```
 
 Example of inline GEN assembly writing to a private global variable:
```

Comment:
From this line to L849 - are not needed after 'commit()' is not needed for private global variables.
Those lines are identical to those above L831-L836

Reply:
yeah good point, i dont think we need a private global example. will remove it now, thx